templates/github/.github/workflows/update_ci.yml.j2

```diff
@@ -34,12 +34,10 @@ jobs:
         with:
           committer: {{  'pulpbot <pulp-infra@redhat.com>' }}
           author: {{  'pulpbot <pulp-infra@redhat.com>' }}
-          branch: ${GITHUB_REF#refs/heads/}-update-ci-files
-          base: ${GITHUB_REF#refs/heads/}
```

Comment:
I read through [the docs](https://github.com/peter-evans/create-pull-request#action-inputs) and it looks like if I omit the inputs it'll do what I want by automatically creating a new branch and using the workflow's base branch.